.travis.yml

```diff
@@ -1,14 +1,6 @@
 language: clojure
-lein: 2.7.1
+lein: 2.9.1
+dist: bionic
 jdk:
   - openjdk8
 script: ./ext/travisci/test.sh
-notifications:
-  email: false
```

Comment:
don't we want to keep the `email: false` bit?

Reply:
I see email false in both puppetserver & tk-jetty9, I think we want to keep it.

Reply:
Ah right, that should be kept. Patchset updated.